src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -1341,31 +1340,31 @@ public enum Dataset
         /// <summary>
         /// The options for the <see cref="ImageClassificationTransformer"/>.
         /// </summary>
-        internal sealed class Options : TransformInputBase
+        public sealed class Options : TransformInputBase
```

Comment:
This shouldn't inherit from `TransformInputBase`. That class is for data transforms, and this is a trainer.

Reply:
done.

---
In reply to: [337694597](https://github.com/dotnet/machinelearning/pull/4362#discussion_r337694597) [](ancestors = 337694597)